README.md

```diff
@@ -599,9 +599,9 @@ usually a Jackson module.
 `master` branch is for developing the next major Jackson version -- 3.0 -- but there
 are active maintenance branches in which much of development happens:
 
-* `2.16` is the branch for "next" minor version to release (as of July 2023)
-* `2.15` is the current stable minor 2.x version
-* `2.14` is for selected backported fixes
+* `2.19` is the branch for "next" minor version to release (as of July 2023)
+* `2.18` is the current stable minor 2.x version
```

Comment:
2.19 as of February 2025 not July 2023

Reply:
Wow I forgot to push the date change 🙉

Thank u for looking out and another PR u created @pjfanning ! 👍🏼👍🏼